app/scripts/modules/core/pipeline/config/templates/pipelineTemplate.service.ts

```diff
@@ -0,0 +1,51 @@
+import {module, IPromise} from 'angular';
+import {$q} from 'ngimport';
```

Comment:
Since you're marking what should be added/uncommented when gate endpoint is created, probably worth marking what should be removed as well? Good for keeping track of cruft for later.